server/opts.go

```diff
@@ -479,6 +479,11 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 				errors = append(errors, err)
 				continue
 			}
+			if dur < 30*time.Second {
+				err := &configErr{tk, fmt.Sprintf("invalid lame_duck_duration of %v, minimum is 30 seconds", dur)}
```

Comment:
We could probably do 1 sec if we have time, and hence exit early if we have small number of clients.

So LB = 60 sec. Grace = 10secs. So 50Sec window, if NumClients < 50, Close 1 every second. Then exit when NumClients == 0.

Reply:
I do max the sleep to ~1sec in this PR per client, so will exit early. What I don't do, is reduce the total time by the grace period. Should I?